packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -27,6 +28,16 @@ export interface FileMetadata {
   url: string;
 }
 
+/**
+ * Strings of {@link _FileDownloadCards} that can be overridden.
+ *
+ * @public
+ */
+export interface FileDownloadCardsStrings {
+  /** Aria label to notify user when focus is on file download button. */
+  downloadFile?: string;
```

Comment:
Could you add an example string here?

Reply:
This string has value "download file" which is being used for Download file button